_posts/articles/2018-11-01-platformos-2of4.md

```diff
@@ -0,0 +1,145 @@
+
+* **Grammar and syntax** correspond to the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html). We didn’t try to reinvent the wheel here, we chose a well-known style guide that works for us.
+* Our documentation consists of different types of topics, like tutorials, concepts, references (e.g. API documentation). To help contribution both for writers and editors, we specified the structure of each content type in **ready-to-use templates**.
+* Contributors are welcome to edit our style guide just like they are welcome to edit any documentation topic.
+
+# Templates
+
+Our site uses Liquid pages, but to make editing easier for contributors, we write documentation content in GitHub Flavored Markdown and use a Markdown converter to turn it into Liquid. Our [templates](https://github.com/mdyd-dev/nearme-documentation/tree/master/marketplace_builder/views/pages/doc-templates) include all non-changeable content, and placeholders with explanations for the parts to add. Placeholders provide information on the recommended format (e.g. title), and any requirements or limitations (e.g. maximum number of characters). Recurring sections, like the question block at the bottom of each page, are added using a partial.  
+
+```ruby
+---
+converter: markdown
+metadata:
+  title: [TITLE, format: VERBing a(n)/the NOUN(s)]
+  description: [short description of the topic, can reuse first sentence, max 300 characters]
+slug: get-started/install-docker-example
+searchable: true
+---
+
+This guide will help you [describe what the user will achieve by the end of the guide]. [Explain goal, domain-related background information, any information that helps understand the purpose or terminology of the tutorial.]
+
+## Requirements
+
+[List of requirements with links (internal or external) and explanations where needed. ]
+
+- [Requirement 1: explanation if applicable]
+- [Requirement 2: explanation if applicable]
+- [Requirement 3: explanation if applicable]
+
+## Steps
+
+[Purpose of the tutorial/title] is a [number of large steps, e.g. two]-step process:
+
+1.  [step 1 short description, format VERB the/a(n)/your NOUN]
+2.  [step 2 short description, format VERB the/a(n)/your NOUN]
+
+### Step 1: [VERB the/a(n)/your NOUN]
+
+[Full description of step with additional content, like code snippet, screenshot, etc.]
+
+### Step 2: [VERB the/a(n)/your NOUN]
+
+### Step 3: [VERB the/a(n)/your NOUN]
+
+## Live example and source code
+[optional: link to live site or source code on GitHub]
+
+## Additional resources
+[optional: bulleted list to additional external resources]
+
+## Next steps
+
+Congratulations! You have [what the user has achieved].
+[Describe what the user can/has to do next.]
+
+- [Link to next tutorial]()
+- [Other link to next tutorial - if applicable]()
+
+{% include 'shared/questions_section' %}
+```
```

Comment:
I need to figure out how to "escape" this include statement... working on it.

Reply:
{% raw %}
stuff with Liquid tags {% include 'for-example' %} you don't want Liquid to interpret
{% endraw %}

Reply:
Thanks! Passes checks now! Yipee.